CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## v1.2.0 [unreleased]
+
+### Release Notes
+
+### Features
+
+### Bugfixes
+
+- [#7554](https://github.com/influxdata/influxdb/pull/7554): update latest dependencies with Godeps.
```

Comment:
I don't consider updating dependencies to be a bug fix. I would add to Features if you're going to put it in the `CHANGELOG`.